fml/command_line.h

```diff
@@ -223,7 +223,14 @@ inline CommandLine CommandLineFromIteratorsWithArgv0(const std::string& argv0,
 }
 
 // Builds a |CommandLine| from the usual argc/argv.
+//
+// First checks if CommandLineFromPlatform returns a value, to avoid using an
+// argv that is not provided as UTF-8.
```

Comment:
tester_main.cc uses this method, and now will pick up the UTF-8 encoded args on windows.

Reply:
I think this will break embedders who provide argc and argv explicitly.

Reply:
Here https://github.com/flutter/engine/blob/e6e94e4ce65290d2c4db7403689a80bc11c2868f/shell/platform/embedder/embedder.cc#L1331

Reply:
It would mean we ignore the specified argc/argv on Windows and instead use the platform based method. I'm not really clear on how that's used today, but I suspect that embedders  on Windows doing this are currently broken unless they're already using this API.

I suppose this could be a problem if the embedder is specifying values for argc/argv that weren't already passed in to the process though...

Reply:
I'll refactor this to a separate method though to avoid disrupting the embedder API.